tests/queries/0_stateless/03202_sz3_codec.sql

```diff
@@ -108,13 +108,6 @@ AND
     c2.key = c1.key - 1
 LIMIT 10;
 
--- This query needs to check that data is really compressed
-SELECT
-    formatReadableSize(sum(data_compressed_bytes)) AS compressed_size,
```

Comment:
Btw, why is it platform dependent?
If it works differently on amd64 and aarch64, why?